### DISCUSSION TITLE
KDE about page location

### DISCUSSION THREAD

**Discussion by Visikde** [SpiralLinux/SpiralLinux-project#471] [Q&A] [answered]
I'm trying to file a bug report & having trouble answering all the questions  
Where do I find the about kde page with actual information?  
When I click on about KDE I get a link to KDE sites, no listing of book worm  
On other KDE installs there is an about page on the bottom of KDE system settings, I'm not seeing one on here  
Discover has some of the answers, I didn't find a program to add that would display the info

**Comment by Visikde** [comment]
My Mageia system setting page I get another category  "System Administration"   I'm not finding info in KDE docs for how to add?
![ScS%Nd-9](https://github.com/user-attachments/assets/eae8cb9a-e937-4673-9831-4cb7da67d5eb)

**Comment by geckolinux** [comment, marked as answer]
Hi there, the `kinfocenter` package might be what you're looking for.

**Comment by Visikde** [comment]
Thank you that is exactly it  
Synaptic fixed me right up !
I just can't say enough good things about Spiral :D 
It's the Debian I've always wanted...

**Reply by geckolinux** [reply]
Really glad to hear you like it.
